static/app/components/hotkeys.tsx

```diff
@@ -0,0 +1,121 @@
+import styled from '@emotion/styled';
+
+import space from 'sentry/styles/space';
+import {getKeyCode} from 'sentry/utils/getKeyCode';
+
+const macModifiers = {
+  18: '⌥',
+  17: '⌃',
+  91: '⌘',
+};
+
+const normalModifiers = {
+  18: 'ALT',
+  17: 'CTRL',
+  91: 'CMD',
+};
+
+const genericGlyphs = {
+  16: '⇧',
+  8: '⌫',
+  37: '←',
+  38: '↑',
+  39: '→',
+  40: '↓',
+  9999: '+',
+};
+
+const keyToDisplay = (
+  key: string,
+  platform?: 'macos' | string
+): [React.ReactNode, boolean] => {
+  let keyStr = key.toUpperCase();
+  let hasMissing = false;
```

Comment:
What does this mean? I see we return a tuple. This function may need a bit better documentation as to what it returns. Alternatively instead of returning a tuple maybe an object would be better since it can be named

Reply:
Ok I see what this is for now.

I think what i would do is have the return look like this

```tsx
{
  label: React.ReactNode;
  /**
   * Indicates if this keybinding is OS specific. E.g. shortcuts that
   * use the `command` key cannot be used on windows.
   */
  specificToOs: 'mac' | 'generic';
}
```